usage-collection/src/main/proto/usage.proto

```diff
@@ -38,6 +38,8 @@ message Event {
         AltusIamUserUpdated altusIamUserUpdated = 6;
         // An Altus user interactively logged in.
         AltusIamUserInteractiveLogin altusIamUserInteractiveLogin = 7;
+        // An Altus user interactively login failure.
```

Comment:
Nit: "An Altus user failed to interactively log in" is better grammatically.

Reply:
Thanks Bill! This is a copy of the file from the thunderhead repo. I will fix it there next time I make an enhancement.

http://ci-cloudbreak.eng.hortonworks.com/job/cloudbreak-pull-request-builder-integration-test/4033/ -> Is it possible to look at what is the failure? I am getting 404. Thanks!

Reply:
Unfortunately, that build aged out of the Jenkins server already. It doesn't take very long. You'll have to update your branch to retrigger the job (a force push should do it, if necessary).